pkg/authenticator/authenticator.go

```diff
@@ -314,3 +318,25 @@ func decodeFromPEM(PEMBlock []byte, publicCert *x509.Certificate, privateKey cry
 
 	return decodedPEM, nil
 }
+
+func getInjectClientCertError(path string) string {
+	// The log file will not exist in old Conjur versions
+	err := utils.VerifyFileExists(path)
+	if err != nil {
+		log.Warn(log.CAKC001W, path)
```

Comment:
I'm really not keen on using just the error codes in the docs - it's hard to correlate a problem to what it prints without having multiple files open and a tiny typo can cause serious issues. It may not be something we can fix now but it should be done differently.

Reply:
i remember having such a conversation. want to start a convo about this in slack? it is out of this PR's scope